examples/crm/src/misc/ImageEditorField.tsx

```diff
@@ -0,0 +1,145 @@
+import {
+    Avatar,
+    AvatarProps,
+    Box,
+    Button,
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogTitle,
+    Stack,
+} from '@mui/material';
+import 'cropperjs/dist/cropper.css';
+import { useFieldValue } from 'ra-core';
+import { createRef, useCallback, useState } from 'react';
+import { FieldProps, Toolbar } from 'react-admin';
+import { Cropper, ReactCropperElement } from 'react-cropper';
+import { useDropzone } from 'react-dropzone';
+import { useFormContext } from 'react-hook-form';
+import { DialogCloseButton } from './DialogCloseButton';
+
+const ImageEditorField = (props: ImageEditorFieldProps) => {
+    const { getValues } = useFormContext();
+    const imageUrl = getValues()[props.source];
+    const [isHovered, setIsHovered] = useState(false);
```

Comment:
**question: ** I think that you want to manage the `change` link on hover using state, isn't it ? If it is the case, maybe you can use a css `:hover` instead ?